src/assets/wise5/themes/default/themeComponents/stepTools/step-tools.component.ts

```diff
@@ -0,0 +1,120 @@
+import { Component, OnInit } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { NodeService } from '../../../../services/nodeService';
+import { ProjectService } from '../../../../services/projectService';
+import { StudentDataService } from '../../../../services/studentDataService';
+
+@Component({
+  selector: 'step-tools',
+  templateUrl: './step-tools.component.html',
+  styleUrls: ['./step-tools.component.scss']
+})
+export class StepToolsComponent implements OnInit {
+  icons: any;
+  idToOrder: any;
+  is_rtl: boolean;
+  nextId: string;
+  nodeId: string;
+  nodeIds: string[];
+  nodeStatus: any;
+  nodeStatuses: any;
+  prevId: string;
+  subscriptions: Subscription = new Subscription();
+  toNodeId: string;
+
+  constructor(
+    private NodeService: NodeService,
+    private ProjectService: ProjectService,
+    private StudentDataService: StudentDataService
```

Comment:
Lower-case local variable names going forward. Change here and other places in the class.

```suggestion
    private nodeService: NodeService,
    private projectService: ProjectService,
    private studentDataService: StudentDataService
```